src/components/factlist.scss

```diff
@@ -0,0 +1,49 @@
+@use "variables" as *;
+
+.facts-wrapper {
+  display: flex;
+  flex-direction: column;
+  margin: 20px;
+}
+
+.facts-header {
+  text-align: center;
+  padding: 0;
+}
+
+.section-header {
+  font-size: 8rem;
+  text-align: center;
+}
+
+.fact-header {
+  font-size: 5rem;
+  text-align: left;
+}
+
+.facts-list {
+  display: grid;
+  row-gap: 1rem;
+  grid-template-columns: auto;
+}
+
+li {
+  list-style: none;
+}
+
+.card {
```

Comment:
This is quite a generic class name, and could easily clash with other `card` elements made by the other pairs! Please give this a more specific name (like `fact-card`)